ipn/ipnlocal/local.go

```diff
@@ -2438,36 +2442,37 @@ func (b *LocalBackend) Ping(ctx context.Context, ip netip.Addr, pingType tailcfg
 	}
 }
 
-func (b *LocalBackend) pingPeerAPI(ctx context.Context, ip netip.Addr) (peer *tailcfg.Node, peerBase string, err error) {
+func (b *LocalBackend) pingPeerAPI(ctx context.Context, ip netip.Addr) (peer tailcfg.NodeView, peerBase string, err error) {
+	var zero tailcfg.NodeView
```

Comment:
I've been calling this noNode/noUser in a few places.

Reply:
I'm getting ready for https://github.com/golang/go/issues/61372 in which cas we just need to delete this declaration line and it'll keep working.